### PR TITLE
LPS-166954-conditions Make Conditions Panel collapsible

### PR DESCRIPTION
**Pull request by Stephy6**
Motivation
=======
This is a PR made after [this](https://github.com/liferay-tango/liferay-portal/pull/3036) one. It makes the Conditions panel (the one under the Scope Card) collapsible.
[LPS-169317](https://issues.liferay.com/browse/LPS-169317)

Proposed Solution
========
The proposed solution consists in making the Conditions panel collapsible by changing the component that renders it from a ClaySheet to a ClayPanel, because the ClaySheet isn't able to collapse.

Steps to Verify:
----------------
1. Go to People -> Segments.
2. Check that the Conditions panel is collapsible.

Screenchots:
----------------
Screenshot 2022-11-23 at 12 44 38
Screenshot 2022-11-23 at 12 44 47

### CODE REVIEW

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_contributor_builder.scss [hunk 1]
```diff
@@ -37,6 +37,10 @@
 				.content-wrapper {
 					padding: 40px 24px;
 
+					.panel-header-link {
```
Comment (claraizquierdo) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_contributor_builder.scss:40]: I would not remove the box shadow if the Clay component already have it.

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js [hunk 2]
```diff
@@ -129,6 +130,12 @@ class ContributorBuilder extends React.Component {
 			'criteria-builder-section-sidebar--with-warning': showDisabledSegmentationAlert,
 		});
 
+		function handleClick(event) {
```
Comment (claraizquierdo) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js:133]: Maybe a more specific name for this function would be better. Something like `handleViewMembersClick`. What do you think?

**Review by claraizquierdo — CHANGES_REQUESTED** [review]
Hi @Stephy6 ,

I left some comments in the code. I also see that the padding right of the Panel title is not the same than in the design:

Captura de pantalla 2022-11-23 a las 17 51 50


Captura de pantalla 2022-11-23 a las 17 53 49

The padding-bottom of the panel-content doesn't look correct either.

I think that we should apply the styles to the `panel-header-link` added in this PR https://github.com/liferay-tango/liferay-portal/pull/3036

If you are planning to pass the other PR first there is no problem, but if not, I would add the styles here too.

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_contributor_builder.scss (hunk 1 above)
Comment (Stephy6) [reply modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_contributor_builder.scss:40]: Done! thank you!

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js (hunk 2 above)
Comment (Stephy6) [reply modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js:133]: Done! thank you!

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js [hunk 3]
```diff
@@ -129,6 +130,12 @@ class ContributorBuilder extends React.Component {
...
+			onPreviewMembers();
+
+			event.stopPropagation();
```
[the remark this answers is not in the document]
Comment (Stephy6) [reply modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js:136]: Done! thank you!

**Comment by Stephy6** [comment]
ci:test:sf

**Comment by Stephy6** [comment]
ci:test:relevant

**Comment by Stephy6** [comment]
ci:forward

**Pull request closed without merging**